payment_bitcoin/i18n/de.po

```diff
@@ -0,0 +1,396 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* payment_bitcoin
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 15.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-02-15 13:08+0000\n"
+"PO-Revision-Date: 2023-02-15 13:08+0000\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: payment_bitcoin
+#: model_terms:ir.ui.view,arch_db:payment_bitcoin.transaction_form_bitcoin
+msgid "&amp;nbsp;"
+msgstr ""
+
+#. module: payment_bitcoin
+#: model:mail.template,body_html:payment_bitcoin.mail_template_data_bit_coin_order_notification
+msgid ""
+"<?xml version=\"1.0\"?>\n"
+"<div style=\"margin: 0px; padding: 0px;\">\n"
+"                <p style=\"margin: 0px; padding: 0px; font-size: 13px;\">\n"
+"                    Dear ,\n"
+"                    <br/>\n"
+"                    <br/>\n"
+"                    The Amount received for Order {{object.name}} is less than actual order amount.Please verify\n"
+"                </p>\n"
+"            </div>\n"
+"        "
+msgstr ""
+
+#. module: payment_bitcoin
+#: model_terms:ir.ui.view,arch_db:payment_bitcoin.view_account_config_min_unused_bitcoin
+msgid ""
+"<span class=\"o_form_label\">Hours</span>\n"
+"                            <span class=\"fa fa-lg fa-building-o\" aria-label=\"Addresses checking for orders which are not older than\" role=\"img\"/>"
+msgstr ""
+
+#. module: payment_bitcoin
+#: model_terms:ir.ui.view,arch_db:payment_bitcoin.payment_bitcoin_acquirer_form
+msgid ""
+"<span>\n"
+"                        <strong>In Minutes</strong>\n"
+"                    </span>"
+msgstr ""
+
+#. module: payment_bitcoin
+#: model:ir.model.fields,help:payment_bitcoin.field_payment_acquirer__deadline
+msgid ""
+"Add deadline to bitcoin payment, within this deadline payment should be done"
```

Comment:
translation missing

Reply:
Better English would be: "Add a deadline to Bitcoin payments within which the payment should be made."

German: "Fügen Sie Bitcoin-Zahlungen eine Frist hinzu, innerhalb derer die Zahlung erfolgen soll."

Reply:
"In Minutes" translates to "In Minuten"